api/test/pact/pact-tests/settings/provider.mock.ts

```diff
@@ -15,7 +15,7 @@ export class PactTestSetup {
     this.provider = new Pact({
       consumer: "xui_webApp",
       dir: path.resolve(process.cwd(), "api/test/pact/pacts"),
-      log: path.resolve(process.cwd(), "api/test/pact/logs", "mockserver-integration.log"),
+      log: path.resolve(process.cwd(), "api/test/pact/logs", config.provider+".log"),
```

Comment:
Are you not missing a double quote at the start of config.provider ?

Reply:
Also, other parts of the project use single quotes. Should be consistent.

Reply:
Just seen what is going on here - please use string interpolation instead

```suggestion
      log: path.resolve(process.cwd(), "api/test/pact/logs", `${config.provider}.log`),
```